fla/models/mom_gated_deltanet/modeling_mom_gated_deltanet.py

```
@@ -0,0 +1,561 @@
+from fla.modules.layernorm import rms_norm_linear
+
+if TYPE_CHECKING:
+    from transformers.processing_utils import Unpack
+
+
+logger = logging.get_logger(__name__)
+
+
+class MomGatedDeltaNetMLP(nn.Module):
+
+    def __init__(
```

Pylint (3.3.7)

[refactor] Too many arguments (7/5)

(R0913)